repository/git.go

```diff
@@ -207,6 +207,18 @@ func (repo *GitRepo) RmConfigs(keyPrefix string) error {
 	return err
 }
 
+//ListRemotes lists all the local remotes
+func (repo *GitRepo) ListRemotes() ([]string, error) {
```

Comment:
This function should be generic, usable for other purpose than just what you want it to do right now. That means in particular:

- you need to deduplicate the remotes (`git remote --verbose` generally list twice each remotes, for fetch and push). If they are identical, you should list them only once.
- you should do more processing of the raw output of the command here, not just split each lines 
- the name of the remote is important as well, instead of returning a `[]string` you could return a map with the name of the remote as the key, and the URL as the value

Reply:
Yeah sure and instead of entering a number we could promt the user to enter the branch name eg myfork or origin.